modules/AlphaWalletFoundation/AlphaWalletFoundation/Transactions/Types/TransactionRow.swift

```diff
@@ -52,7 +52,7 @@ public enum TransactionRow {
     }
 
     public var gas: String { transaction.gas }
-    public var gasPrice: String { transaction.gasPrice }
+    public var gasPrice: String { transaction.gasPrice?.max.description ?? "" }
```

Comment:
Should use `(gasPrice?.max).flatMap { String(describing: $0) }  ?? ""` instead of accessing `.description` right?